### ISSUE TITLE
Guide or better docs when?

### ISSUE THREAD

**Issue by UptightSL** [XboxUnity/AuroraScripts#17]
I've been trying to make a bad library for this for personal use, but I don't know much about actually making it work. I feel like a legit guide or something should be made!

**Comment by Swizzy** [comment]
Not sure a guide is needed tbh - there are plenty of guides explaining how LUA works and plenty of tutorials on how to write code.

Once you have those nailed, all you really should be needing is documentation explaining what functions are available and how they work.

**Comment by UptightSL** [comment]
well a list on what functions do would be great, as some functions are left in the dust.  and a starter guide on how to made a script would be great, as i  imagine some people start or try to start and dont get anywhere.

**Comment by Swizzy** [comment]
Ok, we can write a more detailed description of what each function does probably - but a starter guide is i doubt we'll do as there are basic examples you can look at and need to understand LUA/general development to do stuff, which is beyond the scope of this imho.

**Comment by UptightSL** [comment]
I know it's a dead issue and whatnot but will better documentation ever be worked on? It would be nice to have more in-depth information on all the functions.

**Comment by UptightSL** [comment]
Also what version of Lua is used?

**Comment by Swizzy** [comment]
> I know it's a dead issue and whatnot but will better documentation ever be worked on? It would be nice to have more in-depth information on all the functions.

Eventually, probably - but, most of us have alot of other things that we're doing, there are things in Aurora that's being worked on that has higher priority than improving this documentation.



> Also what version of Lua is used?

It's LUA 5.3 with some additions added to it for the Aurora specific functionality (the functions described here)

**Comment by UptightSL** [comment]
> > I know it's a dead issue and whatnot but will better documentation ever be worked on? It would be nice to have more in-depth information on all the functions.
> 
> Eventually, probably - but, most of us have alot of other things that we're doing, there are things in Aurora that's being worked on that has higher priority than improving this documentation. Feel free to add any documentation you think you can do, or get started on what you know etc.
> 
> > Also what version of Lua is used?
> 
> It's LUA 5.3.3 with some additions added to it for the Aurora specific functionality (the functions described here)

Understandable and will do. 👍

**Comment by jrobiche** [comment]
I began writing some personal documentation for the Lua libraries a while back and only work on it every so often. I have added what I have done so far to the **luaLibraries** branch of my [xbox360-aurora-developer-documentation](https://github.com/jrobiche/xbox360-aurora-developer-documentation/blob/luaLibraries/lua-libraries/overview.md) repository.

I don't think my repository could ever get merged into this repository though. The problem with my repository, aside from my documentation being far from complete, is that I am using it for more than just the Lua documentation. I plan on using it for general documentation relating to Aurora such as the Lua functions, Nova REST API, asset files, and so on. But my repository might become a useful reference once it is closer to being completed.
